cmd/zpool/zpool_main.c

```diff
@@ -51,6 +51,7 @@
 #include <sys/fm/util.h>
 #include <sys/fm/protocol.h>
 #include <sys/zfs_ioctl.h>
+#include <math.h>
```

Comment:
organizationally, you might consider moving all of this pretty-printing code to another file. zpool_main.c is already quite large and contains a lot of different logic. Thus it is becoming unwieldy in its size and scope.